### DISCUSSION TITLE
Dịch linearly separable

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#37] [Terminology] [answered]
Hmm từ này dịch là gì cho hay nhỉ mọi người @mlbvn/handson-ml2 ?

**Comment by cuongvng** [comment]
Khả tách tuyến tính?

**Comment by tiepvupsu** [comment, marked as answer]
Trước trong MLCB anh dịch là [tách biệt tuyến tính](https://github.com/tiepvupsu/ebookMLCB/blob/master/Chapters/05_NeuralNetworks/09_perceptron.tex#L46).

**Reply by ngcthuong** [reply]
Cụm từ này là từ bổ nghĩa cho danh từ đi sau, từ "tách biệt tuyến tính" là danh từ độc lập rồi nên chắc cần từ đi kèm. Cả câu của nó là gì nhỉ?

**Reply by minhduc0711** [reply]
chỉ thế này thôi ạ: "a linearly separable dataset"

**Reply by rootonchair** [reply]
Vậy dịch ra là: `Tập dữ liệu tách biệt tuyến tính`. Mọi người thấy ổn chứ?
